ApplySyntax.sublime-settings

```diff
@@ -153,8 +153,8 @@
         {
             "syntax": "Git Formats/Git Config",
             "rules": [
-                {"file_path": ".*/git/config$"},
-                {"file_path": ".*\\\\git\\\\config$"}
+                {"file_path": ".*/\\.git/config"},
```

Comment:
Why the removal of `$`?

Reply:
So that `/.git/config.bkp` or `/.git/config~` also use that syntax. I deem it very unlikely that someone would name a folder `.git` _and_ have a file and/or tree under it beginning with `config`.

I suppose a reasonable compromise would be something like `".*/\\.git/config[^/]*"` and `".*\\\\\\.git\\\\config[^\\\\]*"` to limit it to files (not folders). If you still want to keep the `$`, that's fine.

Reply:
I guess, if we know there are only 3 configurations, why just target them?  `config(?:\.bkp|~)?$`.  I guess `[^/]*` would be reasonable too, at least it would limit it to the root directory.